src/firmware/app/primitives/move_primitive.c

```diff
@@ -90,19 +92,28 @@ void app_move_primitive_start(TbotsProto_MovePrimitive prim_msg, void* void_stat
     // Change in orientation to reach destination orientation
     const float net_change_in_orientation =
         min_angle_delta(current_orientation, destination_orientation);
+    const float orientation_delta =
+        net_change_in_orientation + (float)revolutions_to_spin * 2.0f * (float)M_PI;
+
+    const float estimated_time_delta = fmaxf(
+        fabsf(distance_to_destination) / (float)(ROBOT_MAX_SPEED_METERS_PER_SECOND),
+        fabsf(net_change_in_orientation) / (float)(ROBOT_MAX_ANG_SPEED_RAD_PER_SECOND));
+
+    // clamp num elements between 3 and TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS
+    const unsigned int num_elements = (unsigned int)fmaxf(
```

Comment:
what's the significance of 3?

Reply:
it's the minimum number of elements, added a comment